pyrasa/utils/aperiodic_utils.py

```diff
@@ -58,85 +60,105 @@ def _get_gof(psd: np.ndarray, psd_pred: np.ndarray, fit_func: str) -> pd.DataFra
 def _compute_slope(
     aperiodic_spectrum: np.ndarray,
     freq: np.ndarray,
-    fit_func: str,
+    fit_func: str | Callable,
```

Comment:
`Callable` is too unspecific for a function that must have a certain set of inputs and outputs to work. Use the protocol pattern as already done [here for the `irasa_fun`](https://github.com/schmidtfa/pyrasa/blob/main/pyrasa/utils/types.py)